packages/core/src/contrib/Queries.ts

```diff
@@ -38,3 +38,24 @@ export const shapeSize = ([t, s]: [string, any]): VarAD => {
     return sqrt(mul(bbox.width, bbox.height));
   }
 };
+
+/**
+ * Return vertices of polygon-like shapes.
+ */
+export const polygonLikePoints = ([t, s]: [string, any]): Pt2[] => {
```

Comment:
I'm planning to remove all the `[t, s]` patterns from queries and constraints. Any reason you left this unchanged? Lmk if there's a known issue. If not, I'll do another pass and type all of them.

Reply:
No good reason. Many helper functions use it, so I just didn't bother to change it. I'm definitely ok with removing it in the future.